sqllineage/core/handlers/source.py

```diff
@@ -72,6 +72,9 @@ def _handle_table(self, token: Token, holder: SubQueryLineageHolder) -> None:
             else:
                 # SELECT * FROM (tab2), which is valid syntax
                 self._handle(token.tokens[1], holder)
```

Comment:
The token is of type Parenthesis. So it goes here first. But in our case, it's not like what's mentioned in the comment. In our case, the value of this token is
```
(VALUES\n     ('data','thing1','1971-01-01 00:00:00', '1971-01-01 00:00:10', '91', 'some text')\n   , ('data','thing2','1971-01-01 00:00:10', '1971-01-01 00:00:20', '92', 'some text')\n   , ('data','thing3','1971-01-01 00:00:10', '1971-01-01 00:00:20', '92', 'some text')\n )
```
It's meaningless to get the index=1 sub_token. We really should add a elif in line #72 to decide whether the index=1 token is VALUE keyword or not.